source/geod24/LocalRest.d

```diff
@@ -808,7 +819,21 @@ public final class RemoteAPI (API) : API
 
         public void shutdown () @trusted
         {
-            C.send(this.childTid, ShutdownCommand());
+            if (Fiber.getThis())
```

Comment:
These three if/else blocks, don't they all do the same thing?

Reply:
It's my mistake. I'll have to modify it.